osm2streets/src/paint.rs

```diff
@@ -0,0 +1,235 @@
+use crate::{marking, marking::Marking};
+
+// We use geom and stay in map space. Output is done in latlon.
+use geom::{Angle, Distance, Line, PolyLine, Polygon, Pt2D, Ring};
+
+use crate::lanes::TrafficMode;
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct PaintArea {
+    // Because I'm lazy and don't want to make different "map space" and "lonlat space" PaintArea
+    // types, I'm using geo::Polygon, so we can just swap out the coords in place. Not ideal.
+    /// A simple ring.
+    pub area: geo::Polygon,
+    pub color: PaintColor,
+}
+impl PaintArea {
+    pub fn new(area: Ring, color: PaintColor) -> Self {
+        Self {
+            area: area.into_polygon().into(),
+            color,
+        }
+    }
+
+    pub fn from(area: Ring) -> Self {
```

Comment:
What if we name this `white_area` or something? White's maybe a generic default for paint in most places, but this feels like a detail that's hard to notice at the call site if you just see `PaintArea::from(ring)`

Reply:
Good call.